app/assets/javascripts/comments.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+$(document).ready(function() {
```

Comment:
Yes, i know about 'remote true', but there was problem with displaying  validation errors, that i couldn`t solve. And I have decided to use AJAX and custom script for showing errors.